### DISCUSSION TITLE
issue in loading Categories

### DISCUSSION THREAD

**Discussion by Maheshdvp999** [learndevopsonline/learndevopsonline#1337] [Q&A] [answered]
Hi team, Hi had created my frontend , categories and mongodb servers and installed with thier respective shell scripts successfully with my catagoriers are not showing could you please help me in that.

frontend ip address : 18.234.57.130 ,172.31.35.155
catagories ip address : 34.238.169.95 , 172.31.40.248
 mongodb ip address: 54.88.118.78, 172.31.38.103

![image](https://github.com/learndevopsonline/learndevopsonline/assets/134056076/d6252fed-4795-47d0-87c6-4b6f46bdf5ac)

**Comment by sreedharm07** [comment]
not able to login to servers
please switch on your servers

**Reply by Maheshdvp999** [reply]
Hi please use this ip addresses 

catalogue 54.82.190.217, 172.31.32.76
mongodb  54.82.36.248 , 172.31.40.175

frontend 174.129.130.139 , 172.31.40.248

**Reply by sreedharm07** [reply]
hi, @Maheshdvp999 roboshop config file is not copied in the location ( /etc/nginx/default.d)
copy config file properly

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/da372b94-346d-459f-920d-11224fa25631)

**Reply by maheshbusireddy** [reply, marked as answer]
Hi @Maheshdvp999 There is a typo error in frontend.sh script 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/32fb5e5b-32f5-40d1-88fd-73d7c8d643c8)

@ **cp ngnix-roboshop.conf /etc/nginx/default.d/roboshop.conf**  not ngnix it's nginx
